config/routes.rb

```diff
@@ -18,497 +20,502 @@
     delete "/sign_out", to: "devise/sessions#destroy"
   end
 
-  require "sidekiq/web"
-  require "sidekiq_unique_jobs/web"
-  require "sidekiq/cron/web"
-
-  authenticated :user, ->(user) { user.tech_admin? } do
-    Sidekiq::Web.set :session_secret, Rails.application.secrets[:secret_key_base]
-    Sidekiq::Web.set :sessions, Rails.application.config.session_options
-    Sidekiq::Web.class_eval do
-      use Rack::Protection, origin_whitelist: [URL.url] # resolve Rack Protection HttpOrigin
+  # [@forem/delightful] - all routes are nested under this optional scope to
+  # begin supporting i18n.
+  scope "(/locale/:locale)", defaults: { locale: nil } do
+    get "/locale/:locale" => "stories#index"
```

Comment:
This file diff is much scarier than it really is. We nested all routes and added these 2 lines (plus some comments above).

There are several routes here that we could move outside of the `/locale/:locale` scope because we don't need them internationalized. I figured by starting with them all nested, we can unnest them over time. As long as the default URLs still work, it doesn't hurt to have them here.

Nesting internationalization routes under `/locale` (open to changing this to `/language` or anything else) makes things cleaner. We don't have to fight so many dynamic top-level routes and we don't have to add further validations to things like organizations, pages, usernames, etc.